spec.md

```diff
@@ -391,11 +391,20 @@ message CreateVolumeRequest {
   // not supported by the Plugin, the call SHALL fail. This field is
   // REQUIRED.
   repeated VolumeCapability volume_capabilities = 4;
-    
+
   // Plugin specific parameters passed in as opaque key-value pairs.
   // This field is OPTIONAL. The Plugin is responsible for parsing and
   // validating these parameters. COs will treat these as opaque.
   map<string, string> parameters = 5;
+
+  // The requested minimal access mode. The Plugin MAY return a volume
+  // that allows a broader access mode, e.g. MULTI_NODE_MULTI_WRITER
+  // when CO asked just for SINGLE_NODE_WRITER. The Plugin MUST NOT
+  // return a volume with more restrictive access mode than the CO
```

Comment:
We need to clearly define what "restrictive" means if we want to go with this route. If we define the order in AccessMode based on "restrictiveness", I don't think that forms a total order (it's a partial order meaning that there might not be an order between two enums).